src/Elastic.Apm/Libraries/Ben.Demystifier/EnhancedStackFrame.cs

```diff
@@ -0,0 +1,93 @@
+﻿// Copyright (c) Ben A Adams. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Diagnostics;
+using System.Reflection;
+
+#nullable enable
+namespace Elastic.Apm.Ben.Demystifier.System.Diagnostics
```

Comment:
I think the namespace for all of these should start `Elastic.Apm.Libraries.Ben.Demystifier`, in line with the convention used for Newtonsoft.Json. Further, for those types in Ben.Demystifier that are placed in `System.*` namespaces, we can remove that part as it won't serve any purpose for our usage.

Reply:
The original `Ben.Demystifier` doesn't have it's own namespace, it's all under `System.Diagnostics` and creates subclasses like `EnhancedStackTrace` derived from `System.Diagnostics.StackTrace`, but leaves it in the same namespace. So it's a bit different compared to Newtonsoft.Json which has its own namespace and we just prefixed it.

Nevertheless I agree with you, best would be to just have this under `Elastic.Apm.Libraries.Ben.Demystifier` - so I moved everything there.